code/index.html

```diff
@@ -4,8 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>WebNN Code Editor</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
-    <link rel="stylesheet" href="./node_modules/codemirror/lib/codemirror.css">
-    <link rel="stylesheet" href="./node_modules/codemirror/theme/railscasts.css">
+    <link rel="stylesheet" href="./libs/codemirror/codemirror.css">
```

Comment:
@huningxin Now it doesn't depend on the **codemirror** node module, so please remove package.json and update the README.md, thanks.

Reply:
Good catch. Added a new commit to solve this issue. Please take another look.